app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::V1::ItemsController < ApplicationController
+  def index
+    @items = Item.all
+    render json: @items
+  end
+
+  def show
+    @item = Item.find(params[:id])
+    render json: @item
+  end
+
+  def new
+    @item = Item.new
+  end
+
+  def edit
+    @item = Item.find(params[:id])
```

Comment:
- [ ] I don't think this is editing `Item` at all, maybe there's something missing here or you aren't using this action at all?

Reply:
I am not using this action at all I was trying to implement that feature but I couldn't and it's also not required